deployments/ansible/README.md

```diff
@@ -24,7 +42,7 @@ Note that this role requires root access.
 ```yaml
 - name: Install Splunk OpenTelemetry Connector
   hosts: all
-  become: yes
+  become_user: root
```

Comment:
Why is this changed?

Reply:
For Windows Its giving a error 'The Powershell family is incompatible with the sudo become plugin' so, changed to 'become_user: root'.